gnovm/pkg/gnolang/helpers.go

```diff
@@ -35,6 +36,11 @@ func IsStdlib(s string) bool {
 	return !strings.HasPrefix(s, "gno.land/")
```

Comment:
For your information, this will become dynamic in #2911, which will be finished soon. 

I recommend waiting for #2911 to be completed and merged before making further hardcoded changes to the domain.

Reply:
agreed